examples/step-13/step-13.cc

```diff
@@ -1129,39 +1093,38 @@ namespace Step13
     // explained in several of the previous example programs, we leave it at
     // that.
     template <int dim>
-    void
-    PrimalSolver<dim>::
-    assemble_rhs (Vector<double> &rhs) const
+    void PrimalSolver<dim>::assemble_rhs(Vector<double> &rhs) const
     {
-      FEValues<dim> fe_values (*this->fe, *this->quadrature,
-                               update_values | update_quadrature_points  |
-                               update_JxW_values);
-
-      const unsigned int   dofs_per_cell = this->fe->dofs_per_cell;
-      const unsigned int   n_q_points    = this->quadrature->size();
-
-      Vector<double>       cell_rhs (dofs_per_cell);
-      std::vector<double>  rhs_values (n_q_points);
-      std::vector<types::global_dof_index> local_dof_indices (dofs_per_cell);
-
-      typename DoFHandler<dim>::active_cell_iterator
-      cell = this->dof_handler.begin_active(),
-      endc = this->dof_handler.end();
-      for (; cell!=endc; ++cell)
+      FEValues<dim> fe_values(*this->fe,
+                              *this->quadrature,
+                              update_values | update_quadrature_points |
+                                update_JxW_values);
+
+      const unsigned int dofs_per_cell = this->fe->dofs_per_cell;
+      const unsigned int n_q_points    = this->quadrature->size();
+
+      Vector<double>                       cell_rhs(dofs_per_cell);
+      std::vector<double>                  rhs_values(n_q_points);
+      std::vector<types::global_dof_index> local_dof_indices(dofs_per_cell);
+
+      typename DoFHandler<dim>::active_cell_iterator cell = this->dof_handler
+                                                              .begin_active(),
+                                                     endc =
+                                                       this->dof_handler.end();
```

Comment:
I wished it would break these four lines differently. It certainly looks odd. But this could be fixed by just giving it a bit more horizontal space.

Reply:
Honestly, I would revise this example step to
```
const auto cell = this->dof_handler.begin_active();
const auto endc = this->dof_handler.end();
```
The original motivation to use a sequence comma here was probably to not rewrite `typename ...` statement a second time. But in terms of readability nothing beats `const auto`.